### PR TITLE
fix: update dependencies to remove security warnings

### PR DESCRIPTION
**Pull request by vonBarbarosa**
fix: update dependencies to remove security warnings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,14 +26,14 @@
     "@semantic-release/git": "9.0.0",
     "browserify": "16.5.2",
     "eslint": "7.31.0",
-    "eslint-config-standardize": "^0.9.1",
-    "eslint-plugin-import": "2.22.0",
+    "eslint-config-standardize": "0.9.1",
+    "eslint-plugin-import": "2.23.4",
```
Comment (victorcsciandt) [inline package.json:30]: Makes sense to keep the semantic versioning here "^"? What do you think?

File: package.json [hunk 2]
```diff
@@ -26,14 +26,14 @@
...
     "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-prettierx": "^0.18.0",
+    "eslint-plugin-prettierx": "0.18.0",
     "eslint-plugin-promise": "5.1.0",
     "husky": "4.3.8",
-    "lint-staged": "11.0.1",
-    "semantic-release": "^17.4.4"
+    "lint-staged": "11.1.1",
+    "semantic-release": "17.4.4"
```
Comment (victorcsciandt) [inline package.json:36]: Makes sense to keep the semantic versioning here "^"? What do you think?

File: package.json [hunk 3]
```diff
@@ -26,14 +26,14 @@
     "@semantic-release/git": "9.0.0",
     "browserify": "16.5.2",
     "eslint": "7.31.0",
-    "eslint-config-standardize": "^0.9.1",
-    "eslint-plugin-import": "2.22.0",
+    "eslint-config-standardize": "0.9.1",
+    "eslint-plugin-import": "2.23.4",
     "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-prettierx": "^0.18.0",
+    "eslint-plugin-prettierx": "0.18.0",
```
Comment (victorcsciandt) [inline package.json:32]: Makes sense to keep the semantic versioning here "^"? What do you think?

File: package.json (hunk 1 above)
Comment (vonBarbarosa) [reply package.json:30]: I agree with that. I don't have write access to the repo, so I'm forking the code to update it.

**Comment by vonBarbarosa** [comment]
I created a new PR from my fork (so I could make the merge etc): https://github.com/digitalmaas/serverless-plugin-browserifier/pull/50
I'm closing this PR.

File: package.json (hunk 1 above)
Comment (vonBarbarosa) [reply package.json:30]: Taking a closer look, actually, the dev dependencies in this repo don't use the `^` but the normal dependencies do. So I guess that's a engineering decision.

**Pull request closed without merging**